canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -110,6 +111,25 @@ export default class PropertiesController {
 		return this.editorSize;
 	}
 
+	setOldParameterDefId(oldParameterDefId) {
```

Comment:
Instead of having these methods can't we just pass the ID in the setForm method?  We can still save the Id but we shouldn't need to keep the old and newIds stored.  We also shouldn't need these methods since only the controller needs to care about the Ids.